src/main/java/competition/subsystems/drive/DriveSubsystem.java

```diff
@@ -19,6 +19,7 @@ public class DriveSubsystem extends BaseDriveSubsystem implements DataFrameRefre
 
     public final XCANMotorController frontLeft;
     public final XCANMotorController frontRight;
+    double motorMultiplier;
```

Comment:
It would also be totally fine to init to the 1.0 here too
```suggestion
    double motorMultiplier = 1.0; // default to full power